src/NuGet.Clients/PackageManagement.VisualStudio/IDE/VSSolutionManager.cs

```diff
@@ -294,6 +293,61 @@ public bool IsSolutionAvailable
             }
         }
 
+        public IEnumerable<string> GetDeferredProjectsFilePath()
```

Comment:
I want this to be executed synchronously which is why I'm calling `JTF.Run` so it doesn't need to be `async`